domrl/agents/big_money_agent.py

```diff
@@ -18,10 +18,31 @@ def find_card_in_decision(decision, card_name):
                 return [idx]
     return [0]
 
+# helper to sort a list of tuples by the second element
+def Sort_List_Of_Tuples(tup):
```

Comment:
I don't think this needs to be a special function. You can just do:

`tup.sort(key = lambda x: x[1]) 
`